debugger/gdb-sgx-plugin/debug_ext.py

```diff
@@ -0,0 +1,306 @@
+import gdb
+import math
+import tempfile
+from typing import List
+
+DEBUG = False
+
+INSTANCE_DEBUG_EXT = None
+
+class ThreadCloneBreakpoint(gdb.Breakpoint):
+    def __init__(self):
+        """
+        Set a breakpoint @ kernel/syscall.c::_SYS_myst_clone
+        Choosing this location because this is where the parent process/thread creates new process/thread
+        We can get the parent pid/tid from here
+        """
+        super(ThreadCloneBreakpoint, self).__init__('kernel/syscall.c:4094', internal=False)
```

Comment:
Line number breakpoints are brittle. Is it possible to choose a function name instead?